Assets/Scripts/BlackHoleConsumption.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+
+public class BlackHoleConsumption : MonoBehaviour
+{
+
+    private LevelManager levelManager;
+    private BlackHolePull blackHolePull;
+
+    private void Start()
+    {
+        levelManager = FindObjectOfType<LevelManager>();
+        blackHolePull = GetComponentInParent<BlackHolePull>();
+    }
+
+    private void OnTriggerEnter(Collider other)
+    {
+        // TODO: only consume when the colliding object is FULLY within the collider
+        var consumedBody = other.gameObject.GetComponent<Rigidbody>();
+        if (consumedBody == null) return;
```

Comment:
I think there's a `HasComponent`...? Or at least there's something better than null checking... or not, but worth checking.

Reply:
Nope, but it looks like it's recommended to do `if (!consumedBody)` instead of `if (consumedBody == null)`, so I'll change it to that